rosidl_generator_py/resource/_msg_support.entry_point.c.em

```diff
@@ -4,15 +4,36 @@
 #include <Python.h>
 #include <stdint.h>
 
-#include <rosidl_generator_c/message_type_support.h>
+@{
+static_includes = {}
+for spec, subfolder in message_specs:
+  if subfolder not in static_includes.keys():
```

Comment:
This is redundant. Even without the check the content of the dict is the same. Therefore I would suggest to remove it for shorter, easier to read code.

Same below.